lib/SearchAndSort/SearchAndSort.js

```diff
@@ -1139,7 +1139,7 @@ class SearchAndSort extends React.Component {
     const isSearchResultCountPresent = isSourceLoaded && !isTotalCountUndefined;
 
     return (
-      <div className={css.delimiter}>
+      <span className={css.delimiter}>
```

Comment:
isn't this affecting consumers?

Reply:
I do not think so.
Tested on Inventory.

The error was "<div> inside <p>".

This div has in CSS nothing but delimiters for it's children
https://github.com/folio-org/stripes-smart-components/blob/3c716765aed08456531042a4bb5054015bc9a624/lib/SearchAndSort/SearchAndSort.css#L17